python/cudf/cudf/core/column/column.py

```diff
@@ -1005,6 +1012,9 @@ def distinct_count(
             raise NotImplementedError(msg)
         return cpp_distinct_count(self, ignore_nulls=dropna)
 
+    def can_cast_safely(self, to_dtype: Dtype) -> bool:
+        return False
```

Comment:
Why's it necessary we have this method in the base class?

Reply:
Oh I see. Yeah, I think `NotImplementedError` is maybe the way to go. I think that follows the pattern of other functions in this class like `binary_operator` which require it to be a subclass to really make sense.